graviti/manager/dataset.py

```diff
@@ -315,47 +432,48 @@ class DatasetManager:
 
     """
 
-    def __init__(self, access_key: str, url: str) -> None:
+    def __init__(self, access_key: str, url: str, owner: str) -> None:
         self._access_key = access_key
         self._url = url
+        self._owner = owner
 
-    def _generate(
-        self, name: Optional[str] = None, offset: int = 0, limit: int = 128
-    ) -> Generator[Dataset, None, int]:
-        response = list_datasets(self._url, self._access_key, name=name, limit=limit, offset=offset)
+    def _generate(self, offset: int = 0, limit: int = 128) -> Generator[Dataset, None, int]:
+        response = list_datasets(self._url, self._access_key, limit=limit, offset=offset)
 
         for item in response["datasets"]:
-            yield Dataset(
-                self._access_key,
-                self._url,
-                item["id"],
-                item["name"],
-                branch_name=item["defaultBranch"],
-                commit_id=item["commitId"],
-                alias=item["alias"],
-            )
+            yield Dataset.from_pyobj(item)
 
         return response["totalCount"]  # type: ignore[no-any-return]
 
     def create(
         self,
         name: str,
         alias: str = "",
-        config_name: Optional[str] = None,
         is_public: bool = False,
+        config: Optional[str] = None,
     ) -> Dataset:
         """Create a Graviti dataset with given name.
 
         Arguments:
             name: The name of the dataset, unique for a user.
             alias: Alias of the dataset, default is "".
-            config_name: The auth storage config name.
             is_public: Whether the dataset is a public dataset.
+            config: The auth storage config name.
 
-        Return:
-            The created :class:`~graviti.dataset.dataset.Dataset` instance.
+        Returns:
+            The created :class:`~graviti.manager.dataset.Dataset` instance.
 
         """
+        arguments: Dict[str, Any] = {
+            "name": name,
+            "alias": alias,
+            "is_public": is_public,
+            "config": config,
+        }
+        response = create_dataset(self._access_key, self._url, **arguments)
+        response.update(arguments)
+
+        return Dataset.from_pyobj(response)
```

Comment:
Why is it necessary to add the method `from_pyobj`? Would `init` do?

Reply:
some response params are different from init